coordinator/src/bin/coordinator.rs

```diff
@@ -218,8 +227,23 @@ async fn main() -> Result<()> {
                             }
                         };
 
+                    // Upon collab closing an expired position we cannot charge a fee using an
+                    // invoice. This dummy hash exists in the database to
+                    // represent zero-amount invoices.
```

Comment:
❓Is this what we want to do, @bonomat? I'm not totally sure after reading [this comment](https://github.com/get10101/10101/issues/909#issuecomment-1623212216).

Reply:
I actually wanted to *try* charging the user in this scenario as well, but this would require a more elaborate process in terms of messages. The coordinator currently triggers the close on the DLC subchannel level. We could change this so the coordinator proposes the close by sending an event over to the app. The app then triggers the close automatically - that way we would tap into the existing process and would charge the fee. I don't see this as part of this PR though. 

My understanding is, that eventually we want to calculate the fee into the payout (and not charge it with a separate invoice); if we do this at some point this problem will go away.

Reply:
Yea, both ways are better then what we have at the moment 😅
It would be nicer if the app would try to collab close an expired position when it detects it cc @holzeis